pkg/types/vsphere/validation/platform.go

```diff
@@ -104,3 +110,29 @@ func validateFolder(p *vsphere.Platform, fldPath *field.Path) field.ErrorList {
 
 	return allErrs
 }
+
+func validateNetwork(p *vsphere.Platform, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
```

Comment:
The only places that use this variable add a single error and return immediately. We may as well do the following instead, for example.
```
return field.ErrorList{field.InternalError(...)}
```